src/legacy/core_plugins/console/public/np_ready/application/components/top_nav_menu.tsx

```diff
@@ -29,15 +29,22 @@ export interface TopNavMenuItem {
 }
 
 interface Props {
+  disabled: boolean;
```

Comment:
Does this always have to be supplied? Would it be OK to have:

```suggestion
  disabled?: boolean;
```

Reply:
It would definitely be ok to have disabled as optional :). This component
just is not used anywhere else at the moment so disabled is always
supplied. But not a good enough reason to not clean up the API 👍